bndtools.m2e/src/bndtools/m2e/BndConfigurator.java

```diff
@@ -237,8 +239,13 @@ private void execJarMojo(final IMavenProjectFacade projectFacade, IProgressMonit
 			SubMonitor progress = SubMonitor.convert(monitor1);
 			MavenProject mavenProject = getMavenProject(projectFacade, progress.newChild(1));
 
-			List<MojoExecution> mojoExecutions = projectFacade.getMojoExecutions("org.apache.maven.plugins",
-				"maven-jar-plugin", monitor1, "jar");
+			List<MojoExecution> mojoExecutions = Stream
```

Comment:
This seems like an awkward way to concat 2 lists. 

```java
List<MojoExecution> mojoExecutions = new ArrayList<>();
mojoExecutions.addAll(projectFacade.getMojoExecutions("org.apache.maven.plugins",
				"maven-jar-plugin", monitor1, "jar"));
mojoExecutions.addAll(projectFacade
						.getMojoExecutions("org.apache.maven.plugins", "maven-jar-plugin", monitor1, "test-jar"));
```

Reply:
yup